handlers/hub_test.go

```diff
@@ -0,0 +1,341 @@
+package handlers_test
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"path"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/ripple-cloud/cloud/data"
+	"github.com/ripple-cloud/cloud/handlers"
+	"github.com/ripple-cloud/cloud/router"
+	"github.com/ripple-cloud/cloud/testhelpers"
+)
+
+func setupServerAddHub(db *sqlx.DB, tokenSecret []byte) (*httptest.Server, error) {
+	r := router.New()
+
+	r.Default(
+		handlers.SetConfig(db, []byte(tokenSecret)),
+	)
+
+	r.GET("/api/v0/hub", handlers.Auth, handlers.AddHub)
+
+	return httptest.NewServer(r), nil
+}
+
+func setupServerShowHub(db *sqlx.DB, tokenSecret []byte) (*httptest.Server, error) {
+	r := router.New()
+
+	r.Default(
+		handlers.SetConfig(db, []byte(tokenSecret)),
+	)
+
+	r.GET("/api/v0/hub", handlers.Auth, handlers.ShowHub)
+
+	return httptest.NewServer(r), nil
+}
+
+func setupServerDeleteHub(db *sqlx.DB, tokenSecret []byte) (*httptest.Server, error) {
+	r := router.New()
+
+	r.Default(
+		handlers.SetConfig(db, []byte(tokenSecret)),
+	)
+
+	r.GET("/api/v0/hub", handlers.Auth, handlers.DeleteHub)
```

Comment:
https://github.com/ripple-cloud/cloud/blob/tests/handlers/hub_test.go#L324

what about this?